css/scss/sections/_index.scss

```diff
@@ -0,0 +1,228 @@
+body {
+  background-color: #307367;
+  @include breakpoint(medium) {
+  background-color: #fff;
+  }
+}
+
+
+p{
+  font-weight: bold;
+color:#fff;
+  @include breakpoint(medium) {
+ color: set-color('grey-700');
+  }
+}
+
+.loginCon {
+ padding: 0;
+
+
+
+    .row {
+    padding:0;
+    }  
+}
+
+.left-log  {
+  padding: 0;
+  margin: 0;
+.bgimg {
+    background-image: url('#{$img-path}/cto-bg2.jpg');
+    height: 100%;
+    min-height: 100vh;
+    background-size: cover;
+}
+}
+
+.right-log {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around; 
+  width: 70%; 
+   @include breakpoint(medium) {
+  justify-content: space-around; 
+  width: 57%; 
+  }
+   @include breakpoint(large) {
+  justify-content: flex-end; 
+  width: 41%;     
+  }
+}
+
+.whiteLogo {
+  margin-top: 80px;
+  width:150px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+
+.vertical-logo img {
+  margin-top: 80px;
+  width:100px; 
+  @include breakpoint(large) {
+  width:150px;       
+  }
+}
+
+.checkbox {
+  font-weight: normal;
+  margin-top: auto;
+  margin-bottom: auto;
+  @include breakpoint(medium) {
+ color: set-color('grey-700');
+     }
+}  
+
+.link {
+  font-weight: normal;
+  margin-top: auto;
+  margin-bottom: auto;
+   color: #fff;
+  @include breakpoint(medium) {
+ color: set-color('grey-700');
+     }
+ }
+.register {
+  .column-small-6{
+width: 27%;    
+padding: 0rem;
+  }
+}
+.button-white {
+ border: 3px solid #307367;
+background: white;
+color: #307367; 
+ height: 50px;
+ width:180px;
+
+  @include breakpoint(medium) {
+  width: 160px; 
+margin-left: 8px;
+}
+  
+  @include breakpoint(large) {
+   width: 180px;
+   margin-left: 8px;
+
+  } 
+}
+
+.button-blue {
+ height: 50px;
+ width: 190px;
+
+  @include breakpoint(small){
+    background: white;
+    color: #307367; }
+  
+  @include breakpoint(medium) {   
+    height: 50px;
+    width: 160px;
+    margin-right: 8px;
+
+@include breakpoint(large) {
+   width: 180px;
+   height: 50px;
+   margin-right: 8px;
+
+
+    }
+}
+}
+
+input[type='username'],
+input[type='password'],
+textarea {
+    display: block;
+    transition: 0.3s ease-in-out;
+    outline: 0;
+    border-bottom: 1px solid rgba(set-color('black'), 0.3);
+    background: transparent;
+    padding: rem-calc(15 0);
+    max-width: 100%;
+    vertical-align: middle;
+    color: set-color('grey-900');
+    font-family: $secondary-font;
+    font-size: rem-calc(16);
+}
+
+.loginForm {
+ margin-top: 0;
+    margin-bottom: rem-calc(30);
+ .form-field-check {
```

Comment:
This is not aligned as the design file.
It should align to the same column as the buttons and the input fields (even tho we don't have it at the moment).
![image](https://user-images.githubusercontent.com/32714864/49344459-6c99df80-f645-11e8-8c21-e98a73abb584.png)